lib/python/ray/worker.py

```diff
@@ -446,8 +449,16 @@ def get_object(self, objectid):
     Args:
       objectid (object_id.ObjectID): The object ID of the value to retrieve.
     """
-    self.plasma_client.fetch([objectid.id()])
-    buff = self.plasma_client.get(objectid.id())
+    buff = self.plasma_client.get_local(objectid.id())
```

Comment:
I don't think reconstruction has to be serialized. We can call reconstruction on a bunch of objects serially, but the local scheduler's event loop should be able to process them asynchronously.